### PR TITLE
chore(forms): remove prop-types

### PR DESCRIPTION
**Pull request by silvioprog**
Closes UXD-1596.

### CODE REVIEW

File: src/components/forms/Input/Input.tsx [hunk 1]
```diff
@@ -71,15 +70,4 @@ const Input = styled.input.attrs<InputProps>(({ isDisabled }) => ({
...
-Input.propTypes = {
-  isInvalid: PropTypes.bool,
-  isDisabled: PropTypes.bool,
-};
-
-Input.defaultProps = {
-  isInvalid: false,
-  isDisabled: false,
-  type: 'text',
-};
```
Comment (ajkl2533) [inline src/components/forms/Input/Input.tsx:83]: @silvioprog `defaultProps` should stay for now, we have different ticket for it and there is migration needed

File: src/components/forms/Select/CreatableSelect.tsx [hunk 2]
```diff
@@ -44,11 +42,6 @@ function CreatableSelect<IsMulti extends boolean = false>({
   );
 }
 
-CreatableSelect.propTypes = {
-  ...Select.propTypes,
-  createNewLabel: PropTypes.string,
-};
-CreatableSelect.defaultProps = Select.defaultProps;
```
Comment (ajkl2533) [inline src/components/forms/Select/CreatableSelect.tsx:51]: Same here, `defaultProps` has to stay

Comment (silvioprog) [reply src/components/forms/Select/CreatableSelect.tsx:51]: Restored.

File: src/components/forms/Select/Select.tsx [hunk 3]
```diff
@@ -30,38 +28,6 @@ function Select<IsMulti extends boolean = false>({
...
-  defaultIsMenuOpen: PropTypes.bool,
-  className: PropTypes.string,
-  menuActions: PropTypes.arrayOf(ActionKindsPropType),
-  maxPillLabelLength: PropTypes.number,
-  maxVisibleItem: PropTypes.number,
-  isMenuPositionRelative: PropTypes.bool,
-  components: PropTypes.shape({}),
-};
-
-Select.defaultProps = {
```
[the remark this answers is not in the document]
Comment (silvioprog) [reply src/components/forms/Select/Select.tsx:56]: Restored.

**Review by ajkl2533 — APPROVED** [review]

**Pull request merged**